cordova-lib/src/plugman/registry/registry.js

```diff
@@ -218,10 +205,18 @@ module.exports = {
 
 /**
  * @method initSettings
+ * @param {Boolean} using npm registry
  * @return {Promise.<Object>} Promised settings.
  */
-function initSettings() {
+function initSettings(npm) {
```

Comment:
nit: `npm` -> `useNpmRegistry`.  Otherwise it looks like a reference to the node module.